Makefile

```diff
@@ -108,107 +94,38 @@ ifeq ($(CONFIG_PROD_MODE),y)
 	export DEPLOY_TEMPLATE_NAME=deploy.tmpl
 	export DEBUG?=False
 else
-	COMPOSE_FILE=docker-compose-dev.yml
+	COMPOSE_FILE=docker-compose.dev.yml
 	export DEPLOY_TEMPLATE_NAME=deploy-dev.tmpl
 	export DEBUG?=True
 endif
 
-
-all: check
-
-build/docker/common/%/$(DUMMY): ##@Build an common image locally
-	$(call build_docker_locally,common,$(BASENAME))
-
-build/docker/agent/%/$(DUMMY): ##@Build an agent image locally
-	$(call build_docker_locally,agent,$(AGENT_BASENAME))
-
-build/docker/%/.push: build/docker/%/$(DUMMY)
-	@docker login \
-		--username=$(DOCKER_HUB_USERNAME) \
-		--password=$(DOCKER_HUB_PASSWORD)
-	@docker push $(BASENAME)-$(patsubst build/docker/%/.push,%,$@):$(IMG_TAG)
-
-docker-common: $(patsubst %,build/docker/common/%/$(DUMMY),$(COMMON_DOCKER_IMAGES)) ##@Generate docker images locally
-
-agent-docker: $(patsubst %,build/docker/agent/%/$(DUMMY),$(AGENT_DOCKER_IMAGES)) ##@Generate docker images locally
-
-docker: docker-common agent-docker
-
-docker-common-%:
-	@$(MAKE) build/docker/common/$*/$(DUMMY)
-
-agent-docker-%:
-	@$(MAKE) build/docker/agent/$*/$(DUMMY)
-
 docker-clean: stop image-clean ##@Clean all existing images
 
-DOCKERHUB_COMMON_IMAGES = api-engine dashboard
-
-dockerhub-common: $(patsubst %,dockerhub-common-%,$(DOCKERHUB_COMMON_IMAGES))  ##@Building latest docker images as hosted in dockerhub
-
-dockerhub-common-%: ##@Building latest images with dockerhub materials, to valid them
-	$(call build_docker_hub,common,$(BASENAME))
-
-DOCKERHUB_AGENT_IMAGES = ansible kubernetes
-
-dockerhub-agent: $(patsubst %,dockerhub-agent-%,$(DOCKERHUB_AGENT_IMAGES))  ##@Building latest docker images as hosted in dockerhub
-
-dockerhub-agent-%: ##@Building latest images with dockerhub materials, to valid them
-	$(call build_docker_hub,agent,$(AGENT_BASENAME))
-
-dockerhub: dockerhub-common dockerhub-agent
-
-dockerhub-pull: ##@Pull service images from dockerhub
-	cd scripts/master_node && bash download_images.sh
-
 license:
 	scripts/check_license.sh
 
-install: $(patsubst %,build/docker/%/.push,$(COMMON_DOCKER_IMAGES))
-
 check: ##@Code Check code format
 	@$(MAKE) license
 	find ./docs -type f -name "*.md" -exec egrep -l " +$$" {} \;
 	cd src/api-engine && tox && cd ${ROOT_PATH}
-	make api-engine
-	make docker-rest-agent
-	make dashboard
+	make docker-compose
 	MODE=dev make start
 	sleep 10
 	# make test-api
 	MODE=dev make stop
 	make check-dashboard
 
-test-case: ##@Code Run test case for flask server
-	@$(MAKE) -C src/operator-dashboard/test/ all
+check-dashboard:
+	docker compose -f tests/dashboard/docker-compose.yml up --abort-on-container-exit || (echo "check dashboard failed $$?"; exit 1)
 
 clean:
```

Comment:
stop: stop the containers using docker-compose
clean: stop + remove containers
deep-clean: clean + remove docker images + clean local db mount path.